practice-one/pages/index.html

```diff
@@ -0,0 +1,605 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="../styles/main.css">
+  <link rel="stylesheet" href="../assets/font-awesome/css/all.css">
+</head>
+<body>
+  <!-- Start Header -->
+  <header class="header section">
+    <div class="header-container container">
+      <div class="header-nav">
```

Comment:
header nav should not include the logo. then the div here is redundant.

Reply:
Thank you for your comments. I will update it